plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -2439,8 +2439,8 @@ private static RecursiveDeleteResult doRecursiveDeleteFiles(FileSystem fileSyste
                 Path filePath = fileStatus.getPath();
                 String fileName = filePath.getName();
                 boolean eligible = false;
-                // never delete presto dot files
-                if (!fileName.startsWith(".presto")) {
+                // never delete Trino dot files
```

Comment:
In any case, it would be good to extend explanation here why it's needed and where such files are being created.

Reply:
Extended comment a bit.